eZ/Publish/API/Repository/Values/Content/Location.php

```diff
@@ -56,6 +58,35 @@
     const STATUS_PUBLISHED = 1;
 
     /**
+     * Map for Location sort fields to their respective SortClauses.
+     *
+     * Those not here (class name/identifier and modified subnode) are
+     * missing/deprecated and will most likely be removed in the future.
+     */
+    const SORT_FIELD_MAP = [
```

Comment:
> (...) I would rather put this in the Core implementation, together with SORT_ORDER_MAP. Method getSortClauses() should then be implemented there as well. (...)

After a quick look - `\eZ\Publish\API\Repository\Values\Content\Location` is extended in a few places:
- `\eZ\Publish\API\Repository\Values\Content\TrashItem`
- `\eZ\Publish\Core\REST\Client\Values\Content\Location`
- `\eZ\Publish\Core\Repository\Values\Content\Location`

so this would require repeating this part or introducing common base class in `Core`. 
Unless you have in mind a different place in `Core` for that.